### ISSUE TITLE
RSI returns NoneType when called from web interface

### ISSUE THREAD

**Issue by TheWalterWhite** [AbenezerMamo/crypto-signal#120]
When attempting comparisons between RSI and integers on the web interface
```
"'>=' not supported between instances of 'NoneType' and 'int'"
```
is thrown. This doesn't happen with CurrentPrice or moving averages. It also seems to happen regardless of the exchange, timeframe or starting point. Default behavior seems to be reporting the RSI values fine though.

**Comment by jrbartola** [comment]
I’ll get to this tomorrow. Thanks for pointing this out!

**Comment by jrbartola** [comment]
Thanks! Should be fixed in PR #121. It should be approved sometime today @TheWalterWhite

**Comment by TheWalterWhite** [comment]
Fixed!
